shared-libs/transitions/test/unit/transitions/mark_for_outbound.js

```diff
@@ -0,0 +1,132 @@
+const rewire = require('rewire');
+const sinon = require('sinon');
+const assert = require('chai').assert;
+
+const markForOutbound = rewire('../../../src/transitions/mark_for_outbound');
+
+describe('mark for outbound', () => {
+  afterEach(() => sinon.restore());
+
+  describe('onMatch', () => {
+    const restores = [];
+    const RECORD_NAME = 'known_record';
+    let mockDb; 
+    let mockConfig; 
+    let mockOutbound;
+    let mockLogger;
+    let change;
+    let clock;
+
+    beforeEach(() => {
+      change = {
+        doc: { type: RECORD_NAME, id: 'RANDOM_UUID' },
+        info: {},
+      };
+
+      mockConfig = {
+        get: sinon.stub(),
+      };
+      
+      restores.push(markForOutbound.__set__('config', mockConfig));
```

Comment:
I think you can do this a simpler way like: 
```suggestion
require('path/to/config')
sinon.stub(config, 'get');
```
Same for db, logger, I believe. Using `rewire` is less readable and is not what we practice most times. I would only use rewire when strictly required. 
Also, keeping the `restores` property is so cumbersome. 
I would rather we redo the loading for every test instead of keeping a list of properties to restore - which is more prone to mistakes. 
```
let markForOutbound;
beforeEach(() => {
  markForOutbound = rewire('../../../src/transitions/mark_for_outbound');
});
```